source/js/hook-duoshuo.js

```diff
@@ -1,46 +1,65 @@
-if (typeof DUOSHUO !== 'undefined') {
-	hook_duoshuo_templates();
-} else {
-	$('#duoshuo-script')[0].onload = hook_duoshuo_templates;
-}
-var is_hook_duoshuo = false;
+if (typeof DUOSHUO !== 'undefined') hookDUOSHUO_tp();
+else $('[src="http://static.duoshuo.com/embed.js"]')[0].onload = hookDUOSHUO_tp;
 
-function hook_duoshuo_templates() {
-	if (is_hook_duoshuo) {
-		return;
-	} else {
-		is_hook_duoshuo = true;
-	}
-	var _D_post = DUOSHUO.templates.post;
-	DUOSHUO.templates.post = function(e, t) {
-		var rs = _D_post(e, t);
-		var is_admin;
-		if (typeof duoshuo_user_ID !== 'undefined') {
-			if (e.post.author.user_id && (e.post.author.user_id == duoshuo_user_ID)) {
-				if(duoshuo_admin_nickname){
-				is_admin = '<span class="this_ua admin">'+duoshuo_admin_nickname+'</span>'
-				}else{
-					is_admin = '<span class="this_ua admin">博主</span>'
-				}
-			} else {
-				is_admin = '';
-			}
-		} else {
-			is_admin = '';
-		}
-		var agent = e.post.agent;
-		if (agent && /^Mozilla/.test(agent)) {
-			rs = rs.replace(/<\/div><p>/, is_admin + show_ua(agent) + '</div><p>')
-		};
-		return rs;
-	}
+function hookDUOSHUO_tp() {
+    var _D_post = DUOSHUO.templates.post
+    DUOSHUO.templates.post = function (e, t) {
+        var rs = _D_post(e, t);
+        var agent = e.post.agent;
+        var isAdmin;
+        if (typeof duoshuo_user_ID !== 'undefined') {
+            if (e.post.author.user_id && (e.post.author.user_id == duoshuo_user_ID)) {
+                if (duoshuo_admin_nickname) {
+                    isAdmin = '<span class="fa">' + duoshuo_admin_nickname + '</span>';

```

Comment:
Line is too long.<br>Identifier 'duoshuo_admin_nickname' is not in camel case.<br>'duoshuo_admin_nickname' is not defined.